pages/reference/base-images/base-images.md

```diff
@@ -10,91 +10,90 @@ excerpt: Docker images maintained by {{ $names.company.lower }}
 
-- `<lang_ver>` is the version of the language stack, for example node.js 10.10, it can also be substituted for `latest`.
-- `<distro_ver>` is the version of the linux distro, for example in the case of Debian there are 4 valid versions, namely `sid`, `jessie`, `buster` and `stretch`.
-- For each combination of distro and stack we have two variants called `run` and `build`. The build variant is much heavier as it has a number of tools preinstalled to help with building source code. You can see an example of the tools that are included in the Debian Stretch variant [here](https://github.com/balena-io-library/base-images/blob/master/balena-base-images/armv7hf/debian/stretch/build/Dockerfile). The `run` variants are stripped down and only include a few useful runtime tools, see an example [here](https://github.com/balena-io-library/base-images/blob/master/balena-base-images/armv7hf/debian/stretch/run/Dockerfile).
-If no variant is specified, the image defaults to `run`
-- <yyyymmdd> the last optional field on tags is the date tag. This is useful for production deployments as these base images are non-moving tags, so no packages in these will update ever.
+- `<lang_ver>` is the version of the language stack, for example, Node.js 10.10, it can also be substituted for `latest`.
+- `<distro_ver>` is the version of the Linux distro, for example in the case of Debian, there are 4 valid versions, namely `sid`, `jessie`, `buster` and `stretch`.
+- For each combination of distro and stack, we have two variants called `run` and `build`. The build variant is much heavier as it has a number of tools preinstalled to help with building source code. You can see an example of the tools that are included in the Debian Stretch variant [here]({{ $links.githubLibrary }}/base-images/blob/master/balena-base-images/armv7hf/debian/stretch/build/Dockerfile). The `run` variants are stripped down and only include a few useful runtime tools, see an example [here]({{ $links.githubLibrary }}/base-images/blob/master/balena-base-images/armv7hf/debian/stretch/run/Dockerfile). If no variant is specified, the image defaults to `run`
+- The last optional field on tags is the date tag `<yyyymmdd>`. This is useful for production deployments as these base images are non-moving tags, so no packages in these will update ever.
 
 #### Examples
 
-`balenalib/raspberrypi3-node:10.10`
+`balenalib/raspberrypi3-node:10.18`
 
-- `<hw>` : raspberrypi3 - The raspberry pi 3 device type.
+- `<hw>` : raspberrypi3 - The Raspberry Pi 3 device type.
 - `<distro>` : omitted, so it defaults to Debian.
 - `<lang>` : node - the Node.js runtime and npm will be installed
-- `<lang_ver>` : 10.10 - This gives us node.js version 10.10.x whatever is the latest patch version provided on balenalib
-- `<distro_ver>` : omitted, so it defaults to `stretch`
+- `<lang_ver>` : 10.18 - This gives us Node.js version 10.18.x whatever is the latest patch version provided on balenalib
```

Comment:
The "Multiple language stacks:" list further up the page lists "Node.js 10.18.0" but here you're using a `<lang_ver>` of `10.18`. Does that imply that `balenalib/raspberrypi3-node:10.18`, `balenalib/raspberrypi3-node:10.18.0` and `balenalib/raspberrypi3-node:10` are all equally valid base-image names? Or is there some other logic at work?

Possibly worthy of more explanation? :man_shrugging:

Reply:
Yes. You can confirm this via `docker pull balenalib/raspberrypi3-node:10.18` and inspecting the output of `docker images` which shows they are all the same image (for this specific example). As I noted earlier I think a few more examples would go a long way for this sort of information.

```
docker images
REPOSITORY                                                         TAG                   IMAGE ID            CREATED             SIZE
balenalib/raspberrypi3-node                                        10                    3bf113fa9176        15 hours ago        231MB
balenalib/raspberrypi3-node                                        10.18                 3bf113fa9176        15 hours ago        231MB
balenalib/raspberrypi3-node                                        10.18.0               3bf113fa9176        15 hours ago        231MB
```

Reply:
Hmmm, I wonder if this _might_ explain the behaviour you discovered at https://github.com/balena-io/docs/pull/1285#discussion_r362953784 where `balenalib/raspberrypi4-64-alpine-python:3-3.11-run` points to an image containing Python 3.6.9 (whereas you'd expect it to contain Python 3.8.0) ...

**_Perhaps_** when `python:3.6.9` got added, it also added the `python:3.6` and `python:3` tags, inadvertently "overwriting" the `python:3` tag which was previously pointing to `python:3.8.0`?

That's just a guess though, and only @nghiant2710 will know for sure.